modules/navigation/nav_agent.cpp

```diff
@@ -61,7 +64,9 @@ void NavAgent::_update_rvo_agent_properties() {
 		rvo_agent_3d.avoidance_layers_ = avoidance_layers;
 		rvo_agent_3d.avoidance_mask_ = avoidance_mask;
 		rvo_agent_3d.avoidance_priority_ = avoidance_priority;
-	} else {
+	} else
+#endif
+	{
```

Comment:
No, because then the closing bracket would be unmatched, it'd be:
```cpp
void NavAgent::_update_rvo_agent_properties() {
		rvo_agent_2d.neighborDist_ = neighbor_distance;
		...
		rvo_agent_2d.avoidance_priority_ = avoidance_priority;
	}
	...
}
```
Which wouldn't be valid

Reply:
Ok must be reading it wrong.  Thanks!